tests/xcm-simulator/contracts/async-xcm-call-no-ce/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "contract"
+version = "0.1.0"
+authors = ["Ashutosh Varma <ashutoshvarma11@live.com>"]
```

Comment:
```suggestion
authors.workspace = true
```

Reply:
Currently ink! contracts does not supports being in a workspace very well. Therefore contracts package is not part of workspace. See the `[workspace]` at the end of this `Cargo.toml` to explicitly make this not a part of outer workspace, otherwise contract build will simply fail.

Reply:
In this case I think we should keep it consistent with the authors in workspace. https://github.com/AstarNetwork/Astar/blob/479a501d72c580261533ec94389862b69e0f5c89/Cargo.toml#L23

Tho not sure why the authors are not *devs* but *devops*.

Reply:
@ashutoshvarma what is the exact issue?

Workspace deps make our life easier, and this feature doesn't seem like something that should break.